<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Customers</class>
 <widget class="QMainWindow" name="Customers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>89</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Customers</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QTableWidget" name="tableCustomers">
      <property name="locale">
       <locale language="French" country="Switzerland"/>
      </property>
      <column>
       <property name="text">
        <string>Id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nom</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Prénom</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Adresse</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>NPA</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Localité</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Téléphone</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Email</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Mobile</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Champ de Recherche</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lblSearchBox"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
